pallets/schemas/src/lib.rs

```diff
@@ -131,5 +147,14 @@ pub mod pallet {
 			);
 			Ok(bounded_fields)
 		}
+
+		pub fn calculate_schema_cost(schema: Vec<u8>) -> FeeDetails<BalanceOf<T>> {
+			// TODO weight to currency/token conversion needed
+			// reference: https://github.com/paritytech/substrate/blob/0ba251c9388452c879bfcca425ada66f1f9bc802/frame/transaction-payment/src/lib.rs#L445
+			// Issue: https://github.com/LibertyDSNP/mrc/issues/77
+			let schema_weight = T::WeightInfo::register_schema(schema.len() as u32);
+			let tip = 0u32.into();
+			FeeDetails { inclusion_fee: None, tip }
```

Comment:
Actual function calculating weights and there will be a converter needed to translate weights in mrc currency 
Have added issue #77

Reply:
can this go in the rpc side of things? Can we flip the dependency? I think all we need from here is the weights for the rpc?

Reply:
One more generic question. If we are just going to use the `len` for calculation does it make sense to just get the data size from user instead of the whole schema? It will reduce used resources on the node but it will also might cause some discrepancy in case the user does not send the correct data len

Reply:
Yes it is done in pallet and at rpc side we want to ensure correct format is being returned because pallet and rpc (off chain) arent connected, rpc will just try to translate before sending it out

Reply:
@aramikm  good question, we might also run validation along with cost calculation when avro no_std is available (if it is available), also its safer to not give away raw numbers for a given input  len only, if someone does not have a schema should not be able to playground with this and we might reject in rpc if schema is not valid